activejob/lib/active_job/arguments.rb

```diff
@@ -90,11 +87,7 @@ def serialize_argument(argument)
           serialize_indifferent_hash(argument)
         when Hash
           symbol_keys = argument.each_key.grep(Symbol).map!(&:to_s)
-          aj_hash_key = if Hash.ruby2_keywords_hash?(argument)
-            RUBY2_KEYWORDS_KEY
-          else
```

Comment:
My original PR removed this side of the conditional. I'm coming to realize that I am unsure of which is the viable post-3.2 branch. I plan on closing this PR, but hope to raise the presence of pre-3.0 Ruby code in this module.